packages/haul-core/assets/debugger.html

```diff
@@ -5,11 +5,11 @@
 <head>
   <meta charset="utf-8">
 
-  <link rel="icon" type="image/png" href="/assets/haul_icon.ico" />
+  <link rel="icon" type="image/png" href="/haul_assets/haul_icon.ico" />
```

Comment:
the reference is changed, but the directory is not

Reply:
Check: https://github.com/callstack/haul/pull/596/files#diff-8f97cf447035a3f3202862e620a9122eL101